### PR TITLE
moving updated MD files from gh-pages branch to vmware/master/doc [SKIP CI]

### PR DESCRIPTION
**Pull request by shuklanirdesh82**
moving updated MD files from gh-pages branch to vmware/master/doc.

This is a final PR to meet objective of #1370

As part of #1498, missing commits from `vmware/master/docs/*.md` merged to `gh-pages/jeckyll-docs/*.md`; so the content  has been in synced. Now gh-pages were having some formatting changes were left out in `vmware/master/docs/*.md` hence this PR.

**Note**: This is PR is to bring changes made in gh-pages which never made to `vmware/master/docs/*.md`. 

After this PR content and format for both `vmware/master/docs` & `gh-pages/` will be in sync, at the release all changes from `vmware/master/docs` copies to `gh-pages/`.

### CODE REVIEW

File: docs/features/tenancy.md [hunk 1]
```diff
@@ -1,8 +1,9 @@
-* IMPORTANT: This is an experimental feature supported only on standalone ESX as of release 0.12. We are seeking feedback/requirements to support mult-tenancy across vSphere Cluster/Datacenter. You can drop us a note at containers@vmware.com.
+---
```
Comment (lipingxue) [inline docs/features/tenancy.md]: Multi-tenancy is not a experimental feature any more?

File: docs/features/tenancy.md [hunk 2]
```diff
@@ -12,28 +13,112 @@ Key attributes of tenancy:
...
+
+```
+#/usr/lib/vmware/vmdkops/bin/vmdkops_admin.py vmgroup ls
+Uuid                                  Name      Description                Default_datastore  VM_list
+------------------------------------  --------  -------------------------  -----------------  -------
+11111111-1111-1111-1111-111111111111  _DEFAULT  This is a default vmgroup
+```
+
+### Default privileges
+When a VM tries to manipulate volumes on a datastore which is not configured, the system will use _DEFAULT privilege, which is created automatically by system post install. This _DEFAULT privilege allows access to ANY datastore by default. An admin can edit or remove this record, thus locking down the functionality to allow access only to explicitly configured datastores.
```
Comment (lipingxue) [inline docs/features/tenancy.md]: We don't have default privileges, instead, we create privileges to "_ALL_DS" and "_VM_DS" for default vmgroup post install.

File: docs/features/tenancy.md [hunk 3]
```diff
@@ -12,28 +13,112 @@ Key attributes of tenancy:
...
+
+### Default datastore
 When a VM addresses the volume using short notation (volume_name, without @datastore), all VMs in this vmgroup will use default datastore to resolve short volume reference (volume_name will actually mean volume_name@default_datastore).
+
```
Comment (lipingxue) [inline docs/features/tenancy.md]: Now, "default_datastore" is must be set when creating a vmgroup.

File: docs/features/tenancy.md [hunk 4]
```diff
@@ -12,28 +13,112 @@ Key attributes of tenancy:
...
 If "default_datastore" is not set for a vmgroup, then datastore where the VM resides will be used as "default_datastore".
 
+## Example
+
```
Comment (lipingxue) [inline docs/features/tenancy.md]: The following example need to be updated. "default_datastore" must be specified when creating a vmgroup.

Comment (lipingxue) [inline docs/features/tenancy.md]: admin-cli.md has the updated info regarding the behavior for multi-tenancy, so tenency.md need to be updated accordingly.

File: docs/user-guide/docker-volume-cli.md [hunk 5]
```diff
@@ -1,100 +1,109 @@
...
-Docker volumes are backed by VMDKs. VMDKs support multiple [types](https://kb.vmware.com/selfservice/microsites/search.do?language=en_US&cmd=displayKC&externalId=1022242)
-
-Currently the following are supported
+##### Disk Modes (attach-as)
+Docker volumes used in vDVS are backed by VMDKs. VMDKs are attached to hosts on which containers are running. These VMDKs can be attached in [different modes.](http://cormachogan.com/2013/04/16/what-are-dependent-independent-disks-persistent-and-non-persisent-modes/)
 
-1. Thick Provision Lazy Zeroed ([zeroedthick]((https://kb.vmware.com/selfservice/microsites/search.do?language=en_US&cmd=displayKC&externalId=1022242)))
-2. Thin Provision ([thin]((https://kb.vmware.com/selfservice/microsites/search.do?language=en_US&cmd=displayKC&externalId=1022242)))
-3. Thick Provision Eager Zeroed ([eagerzeroedthick]((https://kb.vmware.com/selfservice/microsites/search.do?language=en_US&cmd=displayKC&externalId=1022242)))
+1. Persistent mode: In persistent mode the VMDK becomes part of VM the snapshot if a snapshot of the VM is taken while VMDK was attached.
```
Comment (lipingxue) [inline docs/user-guide/docker-volume-cli.md]: typo: "VM the snapshot" -> "the VM snapshot"

File: docs/features/tenancy.md (hunk 1 above)
Comment (shuklanirdesh82) [inline docs/features/tenancy.md]: sorry .. can you elaborate here? what do you mean by "Multi-tenancy is not a experimental feature any more?"

File: docs/features/tenancy.md (hunk 2 above)
Comment (shuklanirdesh82) [inline docs/features/tenancy.md]: In that case `vmware/master/docs/features/tenancy.md` is having redundant information and needs to be removed and we can manage such information at `admin_cli.md` only.

File: docs/features/tenancy.md [hunk 6]
```diff
@@ -12,28 +13,112 @@ Key attributes of tenancy:
...
+Lets consider a sample use case where there are 2 teams – Dev and Test working on Product1. Lets create separate vmgroups (namely Product1Dev and Product2Test) for each of the teams where we can put restriction on datastore consumption.
+
+```
+# /usr/lib/vmware/vmdkops/bin/vmdkops_admin.py vmgroup create --name=Product1Dev
```
Comment (ashahi1) [inline docs/features/tenancy.md]: We should correct this example here - we are not specifying --default-datatsore here.

File: docs/features/tenancy.md [hunk 7]
```diff
@@ -12,28 +13,112 @@ Key attributes of tenancy:
...
+vmgroup 'Product1Dev' is created.  Do not forget to run 'vmgroup vm add' and 'vmgroup access add' commands to enable access control.
+
+#/usr/lib/vmware/vmdkops/bin/vmdkops_admin.py vmgroup create --name=Product1Test
```
Comment (ashahi1) [inline docs/features/tenancy.md]: "Default_datastore" is a required parameter.

File: docs/features/vcenter.md [hunk 8]
```diff
@@ -1,3 +1,5 @@
+###...
```
Comment (ashahi1) [inline docs/features/vcenter.md]: Please remove this if does this does not adds any value.

File: docs/features/tenancy.md (hunk 2 above)
Comment (shuklanirdesh82) [inline docs/features/tenancy.md]: should be taken care by #1536

File: docs/features/tenancy.md (hunk 3 above)
Comment (shuklanirdesh82) [inline docs/features/tenancy.md]: should be taken care by #1536

File: docs/features/tenancy.md (hunk 4 above)
Comment (shuklanirdesh82) [inline docs/features/tenancy.md]: should be taken care by #1536

File: docs/features/tenancy.md (hunk 6 above)
Comment (shuklanirdesh82) [inline docs/features/tenancy.md]: should be taken care by #1536

File: docs/features/tenancy.md (hunk 7 above)
Comment (shuklanirdesh82) [inline docs/features/tenancy.md]: should be taken care by #1536

File: docs/features/vcenter.md (hunk 8 above)
Comment (shuklanirdesh82) [inline docs/features/vcenter.md]: done!

File: docs/user-guide/docker-volume-cli.md (hunk 5 above)
Comment (shuklanirdesh82) [inline docs/user-guide/docker-volume-cli.md]: done!

File: docs/features/tenancy.md (hunk 1 above)
Comment (lipingxue) [inline docs/features/tenancy.md]: It seems that you delete the paragraph that "multi-tenancy is a experimental feature".

Comment (shuklanirdesh82) [inline docs/features/tenancy.md]: let's take care as part of #1536.

**Pull request merged**